.github/workflows/build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build
+on: [push]
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-2022, macos-11, ubuntu-22.04]
+        addrsize: ["64"]
+        include:
+          - os: windows-2022
+            addrsize: "32"
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: secondlife/action-autobuild@v3
+        with:
+          addrsize: ${{ matrix.addrsize }}
+  release:
+    needs: build
+    runs-on: [ubuntu-latest]
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: secondlife/action-autobuild-release@v1
```

Comment:
Ah, this needs to be bumped to v2. https://github.com/secondlife/action-autobuild-release/releases/tag/v2.0.0 Sorry, I don't think this newer version has made it everywhere yet.